### ISSUE TITLE
Why don't you publish this gem on Rubygems?

### ISSUE THREAD

**Issue by luctus** [ClickSend/clicksend-ruby#2]
Hello!

I'd like to understand why you decided to offer the code and let the user build their own gems instead of providing the gem ready to use... I believe I'm missing something.

**Comment by kjlape** [comment]
> I'd like to understand why you decided to offer the code and let the user build their own gems instead of providing the gem ready to use... I believe I'm missing something.

Agreed. This makes me a little concerned that by using the ClickSend service I'll end up maintaining their SDK in my apps. If that is the case, competitors with well maintained libs are starting to look better. 😬 I would feel much better with an explanation, because I'm having a hard time coming up with one that favors ClickSend.

**Comment by acallaghan** [comment]
I am also finding this a bit bizzare - is this the genuine ClickSend Github etc?
